azdo_pipelines/base-pipeline.yml

```diff
@@ -4,20 +4,31 @@ parameters:
 
 steps:
 - script: |
-   flake8 --output-file=$(Build.BinariesDirectory)/flake8_amlservice_testresults.xml aml_service
+   flake8 --output-file=$(Build.BinariesDirectory)/lint_testresults.xml --format junit-xml  
   workingDirectory: '$(Build.SourcesDirectory)'
   displayName: 'Run code quality tests'
   enabled: 'true'
 
-# - script: |
-#    pytest --junitxml=$(Build.BinariesDirectory)/unittest-results.xml $(Build.SourcesDirectory)/Code/tests/unit/${{parameters.pipelineType}}
-#   displayName: 'Run unit tests'
-#   enabled: 'true'
+- task: PublishTestResults@2
```

Comment:
We should have just one PublishTestResults task after both lint & Unit tests which picks up both xml files

Reply:
Ok. Done.